reference/orders.v2.oas2.yml

```diff
@@ -3404,6 +3408,10 @@ components:
           type: string
           example: Towel Type 1
           description: The product name that is shown to merchant in Control Panel.
+        gift_certificate_id:
+          type: integer
```

Comment:
🍹 it should be nullable as well right ?

Reply:
@bc-munjal ^^ should this be nullable?

cc @bc-tgomez @donald-nguyen-bc

Reply:
About ready to merge these changes but waiting on hearing back on the nullable integer @bc-munjal 👍

Reply:
@donald-nguyen-bc : good callout, this needs to be nullable, made that change.

@slsriehl @bc-tgomez thanks for the help, this needs to be nullable, I have made those changes.